### ISSUE TITLE
Desktop dosen't reload after any change on desktop modules

### ISSUE THREAD

**Issue by enuoCM** [wojtkowiak/meteor-desktop#154]
After updated to v0.14.4, when run "meteor --mobile-server=127.0.0.1:3000", it always gets 

```
[meteor-desktop] Initial .desktop version inconsistency found. Files have changed during the build, triggering desktop rebuild.
=> Meteor server restarted
```

Then, any change on desktop moudles, the app doesn't restart automatically.

**Comment by wojtkowiak** [comment]
Sorry for that, was not happening on my machine cause of cache... Fixed in `0.15.0`

**Comment by wojtkowiak** [comment]
Is this ok now?
